endpointmanager/pkg/endpointmanager/postgresql/fhirendpointinfostore_test.go

```diff
@@ -98,17 +114,29 @@ func Test_PersistFHIREndpointInfo(t *testing.T) {
 		t.Errorf("Error adding fhir endpointInfo: %+v", err)
 	}
 
+	// Add endpointInfo1 again but with different requested version
+	// Metadata must be added right now, but after this is fixed remove the metadata insert below:
```

Comment:
I don't know what the standard protocol for handling things that will have to be updated later on a feature branch, but maybe it makes sense to mark this with a TODO so whoever works on the next ticket doesn't miss that this has to be changed?